### ISSUE TITLE
Water is broken on 2-Team Sumo

### ISSUE THREAD

**Issue by AllYourBlocks** [AllYourBlocks/ChunkWars#132]

**Comment by slipcor** [comment]
you probably need to replace the flowing water edges with cave_air before saving it, this way it will work with the waterstop function

**Comment by AllYourBlocks** [comment]
> you probably need to replace the flowing water edges with cave_air before saving it, this way it will work with the waterstop function

Yes, that's the fix. I plan to fix all the sumo issues in one go soon.

**Comment by AllYourBlocks** [comment]
Is affected by #154
